mage_ai/shared/dates.py

```diff
@@ -0,0 +1,18 @@
+import datetime
+import pytz
+
+def compare(date1: datetime, date2: datetime) -> int:
+    if date1 is None or date2 is None:
+        return None
+
+    date1_utc = date1.replace(tzinfo=pytz.UTC)
+    date2_utc = date2.replace(tzinfo=pytz.UTC)
+    date1_str = date1_utc.strftime(format='%Y%m%dT%H%M%S')
```

Comment:
Are we able to compare the datetime objects directly after converting them to UTC?